assets/app/scripts/directives/util.js

```diff
@@ -75,11 +75,11 @@ angular.module('openshiftConsole')
         tag: '=?'
       },
       controller: function($scope, annotationFilter, iconClassFilter) {
-        var icon = $scope.icon = annotationFilter($scope.resource, $scope.tag ? $scope.tag + ".icon" : "icon");
```

Comment:
OK, what about `tags` filter and `iconClass` filter? Leave the optional arg or make copies?